docs/Guides/Serverless.md

```diff
@@ -42,6 +42,17 @@ To integrate with AWS, you have two choices of library:
 
 So you can decide which option is best for you, but you can test both libraries.
 
+**Note**:
+
+Fastify will not allow adding more routes or registering plugins once `ready()` is called. See [here](https://github.com/fastify/fastify/issues/1771#issuecomment-515339364)
```

Comment:
Please do not use a "click here" link. https://granicus.com/blog/why-click-here-links-are-bad/